packages/testing-karma/default-config.js

```diff
@@ -48,6 +48,7 @@ module.exports = config => ({
     require.resolve('karma-snapshot'),
     require.resolve('karma-mocha-snapshot'),
     require.resolve('karma-chrome-launcher'),
+    require.resolve(require.resolve('./src/snapshot-filename-preprocessor.js')),
```

Comment:
why is a nested `require.resolve(require.resolve(` needed?

Reply:
It's not :D I didn't have my morning coffe when I wrote that, I think.